SPECS/R/R.spec

```diff
@@ -2,13 +2,14 @@
 Summary:        A language for data analysis and graphics
 Name:           R
 Version:        4.1.0
-Release:        2%{?dist}
+Release:        3%{?dist}
 License:        GPLv2
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Daemons
 URL:            https://www.r-project.org
 Source0:        https://cran.r-project.org/src/base/R-4/R-%{version}.tar.gz
+Patch0:         0001-configure-fix-compilation-with-curl-8.0.0.patch
```

Comment:
Could you only add a comment explaining that this is our best hunch fix and the patch will need to be removed once `R` "officially" supports cURL 8+?